homeassistant/components/demo/climate.py

```diff
@@ -104,6 +104,7 @@ class DemoClimate(ClimateEntity):
     """Representation of a demo climate device."""
 
     _attr_should_poll = False
+    _attr_translation_key = "demo_climate"
```

Comment:
This might suggest it needs to be domain prefixed, and also maybe change the value of `climate` to make it less confusing with the entity domain.

```suggestion
    _attr_translation_key = "ubercool"
```